test/e2e/README.md

```diff
@@ -0,0 +1,160 @@
+# Use E2E to test itself
+## Introduction
+After updating the features of e2e, you can use the files in the test/e2e/ directory to perform testing for both new and old features of e2e.
+You can perform testing locally. And when you submit a pull request (PR), GitHub Actions will automatically run the tests.
+### Files Structure
+```
+|- test
+  |- e2e
+    |- concurrency
+      |- fail-fast (concurrency & fail-fast mode)
+        |- internal
+          |- expected.yaml 
+          |- verify.yaml (configuration file)
+        |- expected.yaml
+      |- non-fail-fast (concurrency & non-fail-fast mode)
+        |- internal
+          |- expected.yaml
+          |- verify.yaml (configuration file)
+        |- expected.yaml
+    |- non-concurrency 
+      |- fail-fast (non-concurrency & fail-fast mode)
+        |- internal
+          |- expected.yaml
+          |- verify.yaml (configuration file )
+        |- expected.yaml
+      |- non-fail-fast (non-concurrency & non-fail-fast mode)
+        |- internal
+          |- expected.yaml
+          |- verify.yaml (configuration file)
+        |- expected.yaml
+  |- docker-compose.yaml (run a httpbin container, which can return YAML data)
+  |- e2e.yaml (configuration file for the outer infra E2E)
+
+```
+### How it works
+#### Basic flow of SkyWalking Infra E2E
+1. Set up a environment for testing
+2. Spin up the system under test (SUT), prepare necessary dependencies
+3. Trigger or give inputs to SUT, get outputs from SUT
+4. Compare the actual outputs and the expected outputs
+#### Use 'httpbin' to test E2E
+We use the docker container of 'httpbin' as the SUT, which can receive the 'query' of E2E and return YAML data to E2E.After receiving the YAML data from 'httpbin', the E2E will compare the YAML data with the expected YAML file. At last, the E2E will generate a summary of the result.
```

Comment:
In fact, `httpbin` is the SUT for the internal e2e, right? For the external e2e, the SUT is the internal e2e.